tests/ServiceTests/RoutesTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace ServiceTests;
+
+
+use CommonTestClass;
+use yidas\googleMaps\ApiAuth;
+use yidas\googleMaps\ServiceException;
+use yidas\googleMaps\Services;
+
+
+class RoutesTest extends CommonTestClass
+{
+    /**
+     * @throws ServiceException
+     */
+    public function testService(): void
+    {
+        $lib = new Services\Routes(new ApiAuth('test'));
+        $this->assertEquals('https://routes.googleapis.com/directions/v2:computeRoutes', $lib->getPath());
+        $this->assertEquals('POST', $lib->getMethod());
+        $this->assertEquals(null, $lib->getBody());
```

Comment:
assertNull() usully

Small nitpick: also assertSame() with scalar values to make sure the type also mappes IMO